docs/governance/decred-constitution.md

```diff
@@ -1,57 +1,48 @@
 # <img class="dcr-icon" src="/img/dcr-icons/Constitution.svg" /> Decred Constitution
 
-!!! note
-    The constitution was written prior to Decred's launch; some portions of it are no longer relevant and some are yet to be realized. Politeia will be used, when it launches, to propose amendments to the constitution for ratification by Voters.
+* *Decred* (/ˈdi:ˈkred/, /dɪˈkred/, dee-cred) is a secure, adaptable and self-funding cryptocurrency with a system of community-based governance integrated into its blockchain. The project mission is to develop technology for the public benefit, with a primary focus on cryptocurrency technology. *Decred*, as a currency and as a project, is bound by the following set of rules, which include guiding principles, a system of governance, and a funding mechanism. These rules have been established in an effort to create an equitable and sustainable framework within which to achieve *Decred*'s goals.
 
-*Decred* (/ˈdi:ˈkred/, /dɪˈkred/, dee-cred) is an open, progressive, and self-funding cryptocurrency with a system of community-based governance integrated into its blockchain. The project mission is to develop technology for the public benefit, with a primary focus on cryptocurrency technology. *Decred*, as a currency and as a project, is bound by the following set of rules, which include guiding principles, a system of governance, and a funding mechanism. These rules have been established in an effort to create an equitable and sustainable framework within which to achieve *Decred*'s goals.
+  ------
 
----
-
-## Principles
+  ## Principles
 
 * *Free and Open-Source Software* - All software developed as part of *Decred* shall be free and open source-software.
```

Comment:
May want to specifically describe the free and open source (FOSS) license for the software if not already decided.

Reply:
Since the constitution is (currently) non-binding, I would suggest leaving this high-level. Stakeholders may, for instance, decide via Politeia to fund an integration with an exchange. This exchange may require the code to be closed source for legitimate business/security reasons. Is this software "part of Decred"? Arguably not. There is a very strong ethos (and preference among shareholders) for "Decred software" to always be open source. But this could cause a "grey area" in the constitution, which leads to ill-informed or bad-faith social attacks. I think the current general statement is strong if general.

Reply:
What do you mean by "integration with an exchange?"

Reply:
The software licenses are shown in their respective repositories, as is standard. The core software is under a copyfree ISC license but individual sub-projects would I assume be free to choose an alternative FOSS license if there was a more appropriate one for their needs.

Reply:
I see. I would add a specific provision then requiring individual sub-projects to choose the copyfree ISC license or an alternative FOSS license. Simply to make sure that they are required to work with Decred's principles and values in mind.

Let me know your thoughts on this one.